apps/src/code-studio/pd/workshop_dashboard/components/survey_results/facilitator_averages_table.jsx

```diff
@@ -0,0 +1,119 @@
+import React, {PropTypes} from 'react';
+import {Table} from 'react-bootstrap';
+import _ from 'lodash';
+
+const questionOrder = {
+  facilitator_effectiveness: [
+    'overallHow',
+    'duringYour',
+    'forThis54',
+    'howInteresting55',
+    'howOften56',
+    'howComfortable',
+    'howOften',
+  ],
+  teacher_engagement: [
+    'pleaseRate120_0',
+    'pleaseRate120_1',
+    'pleaseRate120_2',
+  ],
+  overall_success: [
+    'iFeel133',
+    'regardingThe_2',
+    'pleaseRate_2',
+    'iWould',
+    'pleaseRate_3'
+  ]
+};
+
+const questionDenominator = {
+  facilitator_effectiveness: 5,
+  teacher_engagement: 5,
+  overall_success: 6
+};
+
+export default class FacilitatorAveragesTable extends React.Component {
+  static propTypes = {
+    facilitatorAverages: PropTypes.object.isRequired,
+    facilitatorId: PropTypes.number.isRequired,
+    facilitatorName: PropTypes.string.isRequired,
+    questions: PropTypes.object.isRequired,
+    courseName: PropTypes.string.isRequired,
+    facilitatorResponseCounts: PropTypes.object.isRequired
+  };
+
+  renderAverage(average, category) {
```

Comment:
what is `average`? is that the sum of all the averages from questions in that category? is there a better name to communicate that?